docs/configuration/agent.md

```diff
@@ -1348,8 +1348,8 @@ As a key:value list:
 <p/>
 To use JSON, pass a JSON-formatted string or use
 <code>--resources=filepath</code> to specify the resources via a file containing
-a JSON-formatted string. 'filepath' can be of the form
-<code>file:///path/to/file</code> or <code>/path/to/file</code>.
+a JSON-formatted string. 'filepath' can only be of the form
```

Comment:
Can you also update here as well?
https://github.com/apache/mesos/blob/master/src/slave/flags.cpp#L76